src/main/java/redis/clients/jedis/BuilderFactory.java

```diff
@@ -16,8 +16,15 @@
   public static final Builder<Double> DOUBLE = new Builder<Double>() {
     @Override
     public Double build(Object data) {
-      String asString = STRING.build(data);
-      return asString == null ? null : Double.valueOf(asString);
+      String string = STRING.build(data);
+      if (string == null) return null;
+      try {
+        return Double.valueOf(string);
+      } catch (NumberFormatException e) {
+        if (string.equals("inf")) return Double.POSITIVE_INFINITY;
```

Comment:
`inf` -> `+inf`

https://redis.io/commands/zrangebyscore

```
Exclusive intervals and infinity

min and max can be -inf and +inf, so that you are not required to know the highest or lowest score in the sorted set to get all elements from or up to a certain score.
```

Reply:
First of all, the one you've mentioned is about request, not response. This PR is about reading Redis' responses.

After using several commands in several ways, I have found that Redis accepts both `+inf` and `inf` as positive infinity in requests. But, in responses, it only sends `inf` as positive infinity. In this PR, I have already added a test where Redis returns positive infinity. Please check that. It definity returned `inf`, otherwise the test wouldn't have passed.

Lastly, if you still want, I can add another condition for `+inf`.

Reply:
Let's add `+inf` as it doesn't hurt and guard against Redis bugfix or regression or so.

Reply:
Done!